client/src/components/PageLayout.tsx

```diff
@@ -75,13 +75,21 @@ export function AdminPageLayout({
   loading,
   children,
   courses,
-}: PropsWithChildren<{ session: Session; title?: string; courseName?: string; loading: boolean; courses: Course[] }>) {
+  styles,
+}: PropsWithChildren<{
+  session: Session;
+  title?: string;
+  courseName?: string;
+  loading: boolean;
+  courses: Course[];
+  styles?: any;
```

Comment:
plz replace `any` with valid type here and in the whole PR

Reply:
@helenakrasnova  please check entire PR, there are lot of `any`. Please fix them all

Reply:
I fixed all my code with `any` type
I didn't fix old code because refactoring existing code is not a part of this task (which is modalData)  
All other cases are fixed

Reply:
usually we try to fix in edited files such legacy problems. if you find it complex, feel free to ignore